backend/loi-service/Letter-Of-Indent (LOI).postman_collection.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Verify if this collection has all the apis. If not, add them.

Reply:
Added create, modify and search apis